commercial.html

```diff
@@ -0,0 +1,196 @@
+    <aside class="show-lg">
+      <div class="sidebar">
+        <ul class="nav">
+          <li class="nav-item">
+            <a href="#about"><i class="icon icon-help"></i></a>
+          </li>
+          <li class="nav-item">
+            <a href="#human"><i class="icon icon-class"></i></a>
+          </li>
+          </li>
+        </ul>
+      </div>
+    </aside>
+    <section class="maintext">
+      <div class="ml-10 mt-10">
+        <div class="anchor" id="about"></div>
+        <article>
+          <h3>Отделение дополнительного платного образования</h3>
+          <div class="divider"></div>
+          <p>Отделение дополнительного платного образования (ОДПО) работает в составе СУНЦ с 1996 года. До 2015 года оно называлось «Отделение с возмещением затрат на обучение» (ОВЗО).</p>
+          <p>Основная задача ОДПО — создать условия для обучения в СУНЦ тем ребятам, кто не прошел по конкурсу на бюджетные места, а также тем учащимся и их родителям, которые рассчитывают на индивидуальный подход в обучении.</p>
+          <p>Для решения этой задачи в ОДПО создан и хорошо зарекомендовал себя с самого начала работы институт освобожденных классных руководителей.</p>
+          <p>
+            Работа педагогического коллектива ОДПО — это:
+            <ul>
+              <li>организация комфортного микроклимата в классе, который создается на 2–3 года;</li>
+              <li>профессиональная помощь в адаптации ребенка к новым для него условиям и требованиям;</li>
+              <li>своевременное выявление и устранение межличностных конфликтов;</li>
+              <li>постоянный и эффективный контроль за успеваемостью и посещаемостью каждого ученика;</li>
+              <li>своевременная помощь ребенку в трудных ситуациях учебного и личностного характера.</li>
+            </ul>
+          </p>
+          <p>Профиль и программы каждого класса соответствуют профилю и программам бюджетных классов и реализуются теми же преподавателями.</p>
+          <p>Учащиеся ОДПО участвуют не только в общелицейских олимпиадах, но и выходят на региональный и российский уровни олимпиад и конференций; они также принимают активное участие во всех культурных мероприятиях СУНЦ.</p>
+          <p>Составной частью образовательной программы ОДПО является система специальных курсов (дополнительных занятий) по предметам, требующим углубленной подготовки для поступления в вузы.</p>
+          <p>Сплоченность ученического (9–11 классы) и мобильность педагогического коллективов позволяют не только решать перечисленные выше задачи, но также обеспечивать функционирование ОДПО в едином образовательном пространстве СУНЦ.</p>
+          <p>За время существования ОДПО с золотыми и серебряными медалями СУНЦ закончили 15 наших выпускников; около 85% выпускников ОДПО поступают на различные факультеты УрФУ.</p>
+          <p>Словом, приходите учиться в ОДПО СУНЦ — не пожалеете!</p>
+          <p>Наш телефон <b>365-44-66</b>, электронная почта <code>a.v.zvonareva@urfu.ru</code>.</p>
```

Comment:
The code text should be black. For that, in `site-section.css`, replace `table code {` with just `code {`